### PR TITLE
docs(website): document custom transformer step implementation

### PR DESCRIPTION
**Pull request by ahaan1984**
Re-implementation of #116, closes issue #47 

Edit: Changes made mostly to incorporate the feedback from #47:
- fixed the broken link to [steps](https://ibis-project.github.io/ibis-ml/reference/steps-outlier.html). 
- Used sentence case and fixed minor spelling errors.
-  Added a brief guideline regarding setting up a virtual environment (can elaborate more on this if required).
- Removed conclusion and added a brief contribution guide at the end.

### CODE REVIEW

**Comment by deepyaman** [comment]
> Re-implementation of #116, closes issue #47

@ahaan1984 Thanks! I have gone ahead and approved the CI workflow (sorry I missed doing this earlier).

I'll review this soon. To help me out, can you update the description to include the high-level changes you made from #116?

File: docs/tutorial/how_to_create_your_own_transformer.qmd [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: "How to create your own transformer"
+description: |
+  This tutorial provides step-by-step guidance for creating your own ibisML transformer in Python.
```
Comment (deepyaman) [inline docs/tutorial/how_to_create_your_own_transformer.qmd:4]:
```suggestion
  This tutorial provides step-by-step guidance for creating your own IbisML transformer in Python.
```

**Review by deepyaman — DISMISSED** [review]
Just a quick comment—can you make sure to use "IbisML" (instead of "ibisML") everyhere?

**Comment by ahaan1984** [comment]
> Just a quick comment—can you make sure to use "IbisML" (instead of "ibisML") everyhere?

Sure, will do so!

**Comment by ahaan1984** [comment]
> > Just a quick comment—can you make sure to use "IbisML" (instead of "ibisML") everyhere?
> 
> Sure, will do so!

**Comment by ahaan1984** [comment]
Have made the suggested changes, do check.

**Comment by deepyaman** [comment]
> Hi. Are there any changes to be made from my end?

Sorry for the delay in getting back to you! I think this is great overall; I will go ahead and merge.

I was working on some general improvements to the content, but it looks like that work was lost when my machine was wiped without any notification; I'll see if I can revisit that at some point in the future.

Regardless, congrats on your first IbisML contribution @ahaan1984!

**Review by deepyaman — APPROVED** [review]

**Comment by ahaan1984** [comment]
Thank you! I look forward to contributing more actively to IbisML in the future!

**Pull request merged**